services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/CryptoProvider.java

```diff
@@ -98,8 +98,7 @@ public PrivateKey getPrivateKey() {
       try (InputStream privateKeyStream = privateKeyResource.getInputStream()) {
         privateKey = getPrivateKeyFromStream(privateKeyStream);
       } catch (IOException e) {
-        logger.error("Failed to load private key from {}", privateKeyPath, e);
-        throw new RuntimeException(e);
+        throw new RuntimeException("Failed to load private key from " + privateKeyPath, e);
```

Comment:
We propose the less generic `UncheckedIOException` which extends `RuntimeException` and also wraps an `IOException`

Reply:
Replaced with UncheckedIOException.